metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/AbstractConnectionBuilder.java

```diff
@@ -0,0 +1,90 @@
+package eu.europeana.enrichment.rest.client;
+
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+
+public abstract class AbstractConnectionBuilder {
```

Comment:
This does not look like a builder and is not used as one, perhaps the terminology is intertwined here..

Reply:
I changed the name of the class just like we discussed